### DISCUSSION TITLE
Separate options to ignore keyboard shortcuts vs. drag-snapping for an app

### DISCUSSION THREAD

**Discussion by jortronQGov** [rxhanson/Rectangle#1266] [Ideas]
[Release v0.70](https://github.com/rxhanson/Rectangle/releases/tag/v0.70) changed the ignore app functionality to ignore drag-snapping in addition to keyboard shortcuts.

I have to ignore my browser, text editor and IDE in Rectangle because of clashes with Rectangle's keyboard shortcuts. Prior to v0.70 I was still able to use drag-snapping to quickly resize and place windows for these apps. 

Any chance these could be separate options?

**Comment by rxhanson** [comment]
You can get back to how it was done prior to v0.70 with the following Terminal command:

`defaults write com.knollsoft.Rectangle ignoreDragSnapToo -int 2`

Then restart the app.

**Reply by jortronQGov** [reply]
Thank you! 

Would be great if this was exposed as an option in the menu or preferences. Possibly on a per-app basis too?
